code/Control/UserDefinedFormController.php

```diff
@@ -257,13 +258,14 @@ public function process($data, $form)
 
             if (!empty($data[$field->Name])) {
                 if (in_array(EditableFileField::class, $field->getClassAncestry())) {
-                    if (!empty($_FILES[$field->Name]['name'])) {
+                    if (is_uploaded_file($_FILES[$field->Name]['tmp_name'])) {
```

Comment:
We will need a way to mock this, which isn't really possible in PHP - I'd suggest that you pull this line out into a protected class method and a mock could be used in tests to override that method's return value.

Reply:
Silverstripe folds the `_FILES` array into `postVars` on a `HTTPRequest`, so perhaps one could check that structure for the desired details.